pkg/flow/isolate.go

```diff
@@ -78,8 +81,7 @@ type ServiceResponse struct {
 
 type functionRequest struct {
 	ActionID  string
-	Iterator  int
-	Workflow  functionWorkflow
+	fnContext enginerefactor.FunctionContext
```

Comment:
I have mentioned in other comments that I think the header should be renamed slightly. We should also rename all structs & helper functions related to it, so they can be renamed from `fnCtx` to `actionCtx`; `enginerefactor.ActionContext`, etc.